### PR TITLE
BZ-2070577: Adding a step to wait for the API server pods to restart

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.6+

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2070577

Link to docs preview:
https://deploy-preview-45583--osdocs.netlify.app/openshift-enterprise/latest/authentication/bound-service-account-tokens.html#bound-sa-tokens-configuring_bound-service-account-tokens

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@s-urbaniak @stlaz for review please!

Preview: https://deploy-preview-45583--osdocs.netlify.app/openshift-enterprise/latest/authentication/bound-service-account-tokens.html#bound-sa-tokens-configuring_bound-service-account-tokens

File: modules/bound-sa-tokens-configuring.adoc [hunk 1]
```diff
@@ -23,15 +23,7 @@ This step is typically not required if the bound tokens are used only within the
...
-If necessary, you can use the following command to manually restart all pods in the cluster. Be aware that running this command causes a service interruption, because it deletes every running pod in every namespace. These pods will automatically restart after they are deleted.
-
-[source,terminal]
-----
-$ for I in $(oc get ns -o jsonpath='{range .items[*]} {.metadata.name}{"\n"} {end}'); \
-      do oc delete pods --all -n $I; \
-      sleep 1; \
-      done
-----
+If necessary, you can manually restart all pods in the cluster so that the holder will request a new bound token.
```
Comment (stlaz) [inline modules/bound-sa-tokens-configuring.adoc:26]: add something like "Before doing so, make sure to wait for all kubernetes-apiserver instances to rollout with your `serviceAccountIssuer` changes."

File: modules/bound-sa-tokens-configuring.adoc [hunk 2]
```diff
@@ -50,6 +42,46 @@ spec:
...
+----
+AllNodesAtLatestRevision
+3 nodes are at revision 12 <1>
+----
+<1> In this example, the latest revision number is `12`.
++
+If the output shows a message similar to one of the following messages, the update is still in progress. Wait a few minutes and try again.
+
+** `3 nodes are at revision 11; 0 nodes have achieved new revision 12`
+** `2 nodes are at revision 11; 1 nodes are at revision 12`
```
Comment (stlaz) [inline modules/bound-sa-tokens-configuring.adoc:68]: we should not rely on any specific messages, they are not an API and can be changed at any moment. I wonder whether there's a better way to describe the wait.

Comment (bergerhoffer) [reply modules/bound-sa-tokens-configuring.adoc:68]: @stlaz I get what you mean, but we do use this method of checking a few other places in the docs now. How about we move forward with this for now, and then can re-evaluate if we have a better way to check that we can update in all places.

**Comment by bergerhoffer** [comment]
@xingxingxia Can you please review these updates?

Preview: https://deploy-preview-45583--osdocs.netlify.app/openshift-enterprise/latest/authentication/bound-service-account-tokens.html#bound-sa-tokens-configuring_bound-service-account-tokens

File: modules/bound-sa-tokens-configuring.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/bound-sa-tokens-configuring.adoc]: Updated!

File: modules/bound-sa-tokens-configuring.adoc [hunk 3]
```diff
@@ -23,15 +23,7 @@ This step is typically not required if the bound tokens are used only within the
...
-If necessary, you can use the following command to manually restart all pods in the cluster. Be aware that running this command causes a service interruption, because it deletes every running pod in every namespace. These pods will automatically restart after they are deleted.
-
-[source,terminal]
-----
-$ for I in $(oc get ns -o jsonpath='{range .items[*]} {.metadata.name}{"\n"} {end}'); \
-      do oc delete pods --all -n $I; \
-      sleep 1; \
-      done
-----
+If necessary, you can manually restart all pods in the cluster so that the holder will request a new bound token. Before doing so, make sure to wait for a new revision of the Kubernetes API server pods to roll out with your service account issuer changes.
```
Comment (tmalove) [inline modules/bound-sa-tokens-configuring.adoc:26]: Change 'make sure' to 'verify' (Red Hat SSG -  https://redhat-documentation.github.io/supplementary-style-guide/#_m)

File: modules/bound-sa-tokens-configuring.adoc (hunk 2 above)
Comment (stlaz) [reply modules/bound-sa-tokens-configuring.adoc:68]: Sure. I am just hoping that noone will ever configure their tooling to be dependent on this very message as that would be wrong.

**Comment by bergerhoffer** [comment]
@xingxingxia Can you review when you get a chance? Thanks!

Preview: https://deploy-preview-45583--osdocs.netlify.app/openshift-enterprise/latest/authentication/bound-service-account-tokens.html#bound-sa-tokens-configuring_bound-service-account-tokens

**Comment by xingxingxia** [comment]
Sorry was taking PTO for days.
/lgtm
Thank you

**Pull request merged**